Makefile

```diff
@@ -0,0 +1,4 @@
+all: $(addsuffix /handout.pdf,$(wildcard */))
+
+%/handout.pdf: %/handout.tex
+	cd $* && lualatex handout.tex
```

Comment:
can we use arara here? it should make sure we call lualatex (or anything else) twice when needed.

Reply:
not important now